### ISSUE TITLE
String with a | in the data

### ISSUE THREAD

**Issue by buzzsawddog** [tools4j/spockito#2]
Again this might be just something I did not think of yet but I have data with a pipe in it and I dont quite know how to use it:

```
  @Test
  @Spockito.Unroll({ 
    "|Input|Location ID|Event ID|",
    "|123|321|123|321|",
    "|123%7c321|123|321|"
    })
  public void testConstructor(String input, BigInteger locationId, BigInteger eventId) {
    ArchiveId id = new ArchiveId(input);

    Assert.assertEquals(locationId, id.getlocationId());
    Assert.assertEquals(eventId, id.getEventId());
  }
```

Will replace the following:

```
  @Test
  public void testConstructorWithPipe() {
    ArchiveId id = new ArchiveId("123|321");

    Assert.assertEquals(BigInteger.valueOf(123), id.getlocationId());
    Assert.assertEquals(BigInteger.valueOf(321), id.getEventId());
  }

  @Test
  public void testConstructorWithUrlDecodedPipe() {
    ArchiveId id = new ArchiveId("123%7c321");

    Assert.assertEquals(BigInteger.valueOf(123), id.getlocationId());
    Assert.assertEquals(BigInteger.valueOf(321), id.getEventId());
  }
```

**Comment by terzerm** [comment]
You can escape pipe characters in the table as follows:
```java
  @Test
  @Spockito.Unroll({ 
    "|Input|Location ID|Event ID|",
    "|123\\|321|123|321|",
    "|123%7c321|123|321|"
    })
  public void testConstructor(String input, BigInteger locationId, BigInteger eventId) {
    ArchiveId id = new ArchiveId(input);

    Assert.assertEquals(locationId, id.getlocationId());
    Assert.assertEquals(eventId, id.getEventId());
  }
```

**Comment by buzzsawddog** [comment]
@terzerm thanks for responding.  There might be a bug here.

I tried the the test with the pipe excaped
![image](https://cloud.githubusercontent.com/assets/3357067/25665015/9ef727e6-2fd9-11e7-8747-d088e4f1a1da.png)

Here in the debugger we see that the escape is still present
![image](https://cloud.githubusercontent.com/assets/3357067/25664956/72ebac94-2fd9-11e7-8032-1fa74efc68b4.png)

And again here.
![image](https://cloud.githubusercontent.com/assets/3357067/25664990/887cb10c-2fd9-11e7-87ed-518a9fc65e57.png)

**Comment by buzzsawddog** [comment]
@terzerm I created a pull request showing the issue: https://github.com/tools4j/spockito/pull/3

**Comment by terzerm** [comment]
Sorry I completely missed to remove the escaping characters from strings. 

Will fix for release 1.4.

**Comment by terzerm** [comment]
Fixed as per V1.4

**Comment by buzzsawddog** [comment]
@terzerm thanks that worked nicely.
